pwndbg/commands/heap.py

```diff
@@ -549,49 +549,96 @@ def tcachebins(addr=None, verbose=False):
 
 
 parser = argparse.ArgumentParser()
-parser.description = "Find candidate fake fast chunks overlapping the specified address."
+parser.description = "Find candidate fake fast or tcache chunks overlapping the specified address."
 parser.add_argument("addr", type=int, help="Address of the word-sized value to overlap.")
-parser.add_argument("size", nargs="?", type=int, default=None, help="Size of fake chunks to find.")
+parser.add_argument(
+    "size", nargs="?", type=int, default=None, help="Maximum size of fake chunks to find."
+)
+parser.add_argument(
+    "--align",
+    "-a",
+    action="store_true",
+    default=False,
+    help="Whether the fake chunk must be aligned to MALLOC_ALIGNMENT.",
```

Comment:
Maybe that is a stupid question, but when does it have to be aligned or not? Does this depend on libc?

Reply:
I think tcache chunks are the only ones that need to be aligned, otherwise all the checks between the two are pretty much the same.

Reply:
Safe Linking enforces alignment for singly-linked bins (both tcache & fastbins)

Reply:
Can we either add a comment on this or describe this in a docstring?